src/Core/Entities/OrganizationDomain.cs

```diff
@@ -18,19 +18,14 @@ public class OrganizationDomain : ITableObject<Guid>
 
     public void SetNextRunDate()
     {
-        if (JobRunCount == 3)
-        {
-            return;
-        }
```

Comment:
Is this removed because it isn't needed as it doesn't even get to this function if it has run 3 times as determined in `GetManyByNextRunDateAsync()` and the `OrganizationDomain_ReadByNextRunDate` procedure?

Reply:
Yep, that looks like that is it!